packages/account/src/Containers/account.jsx

```diff
@@ -55,20 +53,9 @@ const Account = ({
     const onClickClose = React.useCallback(() => routeBackInApp(history), [routeBackInApp, history]);
 
     React.useEffect(() => {
-        if (should_allow_authentication) {
-            setIsLoading(false);
-        }
         toggleAccount(true);
     }, [should_allow_authentication, toggleAccount]);
```

Comment:
Please remove `should_allow_authentication` from dependencies

Reply:
Sure, I was hesitating to remove this..